backend/db/db.go

```diff
@@ -0,0 +1,96 @@
+// Package db provides a simple way to communicate with database
+package db
+
+// TODO: if necessary, covert the db struct to a Env struct
+// that stores environment of the whole server, e.g. *DB, port, hostname, config, etc.
+// Env will be constructed in main() and passed to all other handlers
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	_ "github.com/go-sql-driver/mysql" // import mysql driver
+	"gopkg.in/ini.v1"
+)
+
+// DatabaseStruct stores a DB* and a slice of tables names
+type DatabaseStruct struct {
+	db         *sql.DB
+	tableNames []string
+}
+
+// New returns a pointer to a DatabaseStruct
+func New(d *sql.DB, t []string) (*DatabaseStruct, error) {
+	return &DatabaseStruct{db: d, tableNames: t}, nil
+}
+
+// NewConnect returns a pointer to a DatabaseStruct while trying to connect using parameters
+func NewConnect(user, password, endpoint, database string, tableNames []string) (*DatabaseStruct, error) {
```

Comment:
change function name to Connect() or perhaps NewConnection() if you really want the new there